lib/liquid/block_body.rb

```diff
@@ -68,7 +68,7 @@ def freeze
 
     # @api private
     def self.unknown_tag_in_liquid_tag(tag, parse_context)
-      Block.raise_unknown_tag(tag, 'liquid', '%}', parse_context)
+      Block.raise_unknown_tag(tag, 'liquid', '%}', parse_context, false)
```

Comment:
`raise_unknown_tag` is a class method. We need a way to pass down `supports_end_tag` so that the error message for using an `end` in a `liquid` tag is the following:
```syntax error (line 3): 'end' is not a valid delimiter for liquid tags. use %}```

and not
```syntax error (line 3): 'end' is not a valid delimiter for liquid tags. use end or %}```

(which would be invalid).

Reply:
It's not the cleanest code, but I wanted to get alignment first before I spend my time on making this super clean 🙃